src/Microsoft.SqlTools.ServiceLayer/Connection/ReliableConnection/ReliableConnectionHelper.cs

```diff
@@ -806,6 +817,23 @@ public static ServerInfo GetServerVersion(IDbConnection connection)
                     serverInfo.OsVersion = reader[0].ToString();
                 });
 
+                int? isBigDataCluster = serverInfo.Options[ServerInfo.OptionIsBigDataCluster] as int?;
+                if (isBigDataCluster != null && isBigDataCluster.Value == 1)
+                {
+                    ExecuteReader(
+                    connection,
+                    SqlConnectionHelperScripts.GetBigDataClusterEndpoints,
+                    delegate (IDataReader reader)
+                    {
+                        List<BigDataClusterEndpoint> bigDataClusterEndpoints = new List<BigDataClusterEndpoint>();
```

Comment:
Minor: I think we can just use `clusterEndpoints` everywhere. It's a lot shorter and has the same basic meaning.